nano/node/node.hpp

```diff
@@ -151,6 +152,8 @@ class active_transactions
 	static unsigned constexpr announcement_long = 20;
 	static size_t constexpr election_history_size = 2048;
 	static size_t constexpr max_broadcast_queue = 1000;
+	std::atomic<uint64_t> active_difficulty;
+	boost::circular_buffer<uint64_t> difficulty_cb;
```

Comment:
This member is declared before started/stopped in the class but is initialized after them in the constructor member initialization list. This will generate warnings on some compilers as they should be in the same order. Easiest way to prevent this (and save duplication/extra lines) is to just initialize default values in-class.